src/frontend/pages/consumers/ProductPage/ProductPage.jsx

```diff
@@ -23,14 +25,25 @@ const characteristic = [
   },
 ];
 
-const ProductPage = () => {
+const ProductPage = ({ params }) => {
+  const [data, setData] = useState([]);
```

Comment:
I forgot, there should be:
```javascript
const [data, setData] = useState(null);
``` 
because `data` withing `{data && (<div>....</div>)}` will be `true`, because, `[]` evaluates like `true`

Reply:
Fixed. It works.